### DISCUSSION TITLE
Clear command, delete item with lore and name.

### DISCUSSION THREAD

**Discussion by WolverStones** [Zrips/CMI#4481] [Feature Requests]
I would welcome the option to delete items lore or name

/clear

**Comment by CloudeLecaw** [comment]
Also it would be nice to be able to clear an item if it has a special unwanted enchant.

command could look like:
cmi clear (playerName) (item(-amount)) **_(-n:name) (-l:loreword) (-e:enchant(-lvl_))** (-s) (+clearType)
